sidebars.js

```diff
@@ -161,40 +314,132 @@ const sidebars = {
               label: 'Getting Started',
               collapsible: true,
               items: [
-                'helm-charts/getting-started-scalar-helm-charts',
-                'helm-charts/getting-started-scalardl-ledger',
-                'helm-charts/getting-started-scalardl-auditor',
-                'helm-charts/getting-started-scalardl-auditor-tls',
-                'helm-charts/getting-started-scalardl-auditor-tls-cert-manager',
-                'helm-charts/getting-started-monitoring',
-                'helm-charts/getting-started-logging',
-                'helm-charts/getting-started-scalar-manager',
+                {
+                  type: 'doc',
+                  id: 'helm-charts/getting-started-scalar-helm-charts',
+                  label: 'Scalar Helm Charts by Using a Helm Chart',
```

Comment:
Seems wordy. How about this?

```suggestion
                  label: 'Use Scalar Helm Charts',
```